### DISCUSSION TITLE
How to estimate “mucosa permeability on basolateral side”

### DISCUSSION THREAD

**Discussion by balamo98** [Open-Systems-Pharmacology/Forum#1429] [Questions & Problems] [answered]
Deer team，
I tried to use the verapamil model which is published in the OSP-model，but I couldn't know some details.
![图片](https://user-images.githubusercontent.com/96385989/210837725-152b70da-2cd5-407d-987f-e2747fe6d7c0.png)
like this picture，I dont know where to estimate “mucosa permeability on basolateral side” in PK-sim，
![图片](https://user-images.githubusercontent.com/96385989/210838399-1b346cfc-18cc-4814-bc10-2fe6641ed64d.png)
I couldn't find the parameter  in “Automated Parameter Identification”.
Is it needs to use Mobi？

Thanks
Yours colin

**Comment by Yuri05** [comment, marked as answer]
Switch to the advanced mode:

![grafik](https://user-images.githubusercontent.com/25061876/210853976-4c723160-60fe-41cc-9359-480d7c27d612.png)

**Reply by balamo98** [reply]
Thank you for your reply, which has given me great help!
